### ISSUE TITLE
A player cannot throw a potion several times

### ISSUE THREAD

**Issue by yoraze** [pmmp/PocketMine-MP#3301]
### Issue description

- Expected result: A player can throw a potion several times
- Actual result: Look at issue title

### Steps to reproduce the issue
1. Join to the server
2. Get a throwable potion from creative inventory
3. Hold to screen... Player throws only one potion, no more...

### OS and versions
* PocketMine-MP: 4.0.0 (commit 17fc49db8)
* PHP: 7.3.13
* Server OS: Linux
* Game version: PE

### Plugins
Bug works without plugins

### Maybe useful notes
I think an error is born here: https://github.com/pmmp/PocketMine-MP/blob/master/src/network/mcpe/handler/InGamePacketHandler.php#L313
Condition is bit invalid: If the player starts throwing the potion many times (hold screen) - at this point the function will **return true** and the player will not be able to drop the potion **more than once**.

I also want to note that **I fixed this bug** by slightly changing this condition:
```php
	if($this->player->isUsingItem() and $this->player->getInventory()->getItemInHand() instanceof Consumable and !$this->player->consumeHeldItem()){
		$this->session->getInvManager()->syncSlot($this->player->getInventory(), $this->player->getInventory()->getHeldItemIndex());
		return true;
	}
```

**Comment by dktapps** [comment]
This is related to the arms-stuck-in-air bug. The flag for using-item is set in wrong conditions.

This problem exists both on stable and master, but a lucky accident on stable stops this bug from manifesting (MobEquipmentPacket unconditionally stops item use on stable).
On master, the packet is ignored when the origin and target hotbar slots are the same, hence why this bug shows.

**Comment by dktapps** [comment]
This is fixed by #3664.
